src/libgcl/libgcl.h

```diff
@@ -169,7 +169,7 @@ enum GCLOperators
     eEqual = 20,
 };
 
-#define GAME_FLAG_BIT_01 0x1
+#define CHAFF_G_ACTIVE 0x1
```

Comment:
These should be moved to the STATE_* flags in game.h if possible

Reply:
Ok, but I can't fix it until this weekend.

Reply:
Ready for merge if you wish.